LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2019, The Polymer Authors. All rights reserved.
```

Comment:
2020?

Reply:
the code here was written in 2019, right?